hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneClusterImpl.java

```diff
@@ -396,6 +401,15 @@ private void waitForHddsDatanodeToStop(DatanodeDetails dn)
     }, 1000, waitForClusterToBeReadyTimeout);
   }
 
+  private static void overrideDatanodeVersions(int dnInitialVersion, int dnCurrentVersion) {
+    if (dnInitialVersion != DatanodeVersion.FUTURE_VERSION.toProtoValue()) {
+      mockDNStatic.when(HddsDatanodeService::getDefaultInitialVersion).thenReturn(dnInitialVersion);
+    }
+    if (dnCurrentVersion != DatanodeVersion.FUTURE_VERSION.toProtoValue()) {
+      mockDNStatic.when(HddsDatanodeService::getDefaultCurrentVersion).thenReturn(dnCurrentVersion);
```

Comment:
In this case the "not future version" check makes sense, because the datanode will never return FUTURE_VERSION directly to the client. Can we add a comment here explaining this since it may not be intuitive for someone looking at this for the first time?